src/commands/env/list.ts

```diff
@@ -55,21 +55,20 @@ export default class EnvList extends SfCommand {
   public retrieveRemoteEnvironments(): DisplayEnvironment[] {
     // No server interactions in this experiement, so use a fake remote env list stored locally in the auth file.
     return this.accounts.entries().reduce((list, [name, account]) => {
-      const type = name === 'heroku' ? 'compute' : 'org';
-      const accountEnvironments = account.environments.map((environmentsName) =>
-        this.convertEnvironment(environmentsName, {
+      const type = name === 'heroku' || name === 'functions' ? 'compute' : 'org';
+      const accountEnvironments = account.environments.map((environmentsName) => {
+        let context = 'sandbox';
+
+        if (name === 'heroku') context = 'heroku app';
+        if (name === 'functions') context = 'functions';
+        if (name === 'org' && environmentsName.includes('scratch')) context = 'scratch';
```

Comment:
This is 100% just stylistic but I like how it reads better and that is makes intent more explicit. Feel free to ignore me if you disagree though :)

```suggestion
        let context;

        if (name === 'heroku') {
            context = 'heroku app';
      	} else if (name === 'functions') {
      	    context = 'functions';
      	} else if (name === 'org' && environmentsName.includes('scratch')) {
      	    context = 'scratch';
      	} else {
      	    context = 'sandbox';
      	}
```

Reply:
I don't care, so sure. Although formatting is off on the suggestion so I will do in a followup.